at-tanvo/D3-Exercise3.rb

```diff
@@ -0,0 +1,13 @@
+puts "Nhap phan tu mang :"
+a = gets.to_i
+arr = []
+for i in 1..a
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/atruby-team/internship-18-summer/blob/7587b87b7304d151b3838535fcf67736f443399f/at-tanvo/D3-Exercise3.rb#L4"></span>Lint/UselessAssignment: Useless assignment to variable - <code>i</code>. (https://github.com/rubocop-hq/ruby-style-guide#underscore-unused-vars)</td>
    </tr>
  </tbody>
</table>